.woodpecker.yml

```diff
@@ -1,7 +1,43 @@
 variables:
-  - &rust_image "rustlang/rust:nightly"
+  - &rust_image "rust:latest"
   - &install_binstall "wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz && tar -xvf cargo-binstall-x86_64-unknown-linux-musl.tgz && cp cargo-binstall /usr/local/cargo/bin"
 
+services:
+  lemmy:
+    name: lemmy
+    image: dessalines/lemmy:0.19.4-beta.2
+    commands:
+      - |-
+        printf '{
```

Comment:
Add this as a file somewhere, probably in your new test folder.

Reply:
the woodpecker pipeline would need to be privileged to map a file into the service which is why this workaround is needed if i understand correctly

Reply:
See L27, you should be able to copy or symlink the file into the correct location, and it should work. All the files should be accessible to the images.

Reply:
i think i tried this in this commit

https://github.com/LemmyNet/lemmy-ui-leptos/pull/54/commits/d70a9b39d5ab9a0b89d7f406952c8685d099d3aa

and the pipeline refused to run because of permissions but i will give it another go

Reply:
here's the documentation that i think applies if i'm getting this right:

https://woodpecker-ci.org/docs/usage/volumes

Reply:
oh sorry i think i understand now with pipelines the src is in the container file system! will see if this works with services

Reply:
ok this is done, thx @dessalines 

just waiting for the pipeline's network issues to recover to get a good build